src/main/resources/assets/cctweaks/lua/prebios-string.lua

```diff
@@ -3,12 +3,28 @@ local function fail(msg)
 	term.setCursorBlink(false)
 	term.clear()
 	term.setCursorPos(1, 1)
-
+	local x, y = term.getSize()
 	-- Print some red text
 	if term.isColour() then term.setTextColour(16384) end
-	term.write(msg)
-
-	term.setCursorPos(1, 2)
+	
+	--Split the message into multiple parts if it's too long for the screen
```

Comment:
I'd rather use the implementation given in `write` if anything, as that splits on words as well.